internal/flink/command_artifact.go

```diff
@@ -0,0 +1,63 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	connectcustompluginv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type customPluginSerializedOut struct {
+	Name           string `serialized:"name"`
+	Id             string `serialized:"id"`
+	ConnectorClass string `serialized:"connector_class"`
+	ContentFormat  string `serialized:"connector_class"`
+	Scope          string
+}
+
+type customPluginHumanOut struct {
+	Name           string `human:"Name"`
+	Id             string `human:"Plugin ID"`
+	ConnectorClass string `human:"Version ID"`
```

Comment:
Why is this called "ConnectorClass" but prints as "Version ID"? Are those the same thing?

Reply:
@brianstrauch , this is a good question. 

we will reuse existing custom connector plugin api for flink artifact. This is just for EA only. Post EA, flink team should propose finalized public api for flink artifact management. 

In order to reuse custom connector plugin api to fit flink EA scenario, we return `versionid` from `connectClass` field for flink-udf case. This will change when Flink GA'ed.

Reply:
Lets at least add some TODOs for all these rough edges until then please

Reply:
@pgaref do you have workitem to create flink artifact api post EA? i can link them here otherwise i'll create one

Reply:
Lets use https://confluentinc.atlassian.net/browse/FRT-334

Reply:
Should the serialized output also say "version_id"?